manager/database/mysql.go

```diff
@@ -80,17 +80,19 @@ func formatDSN(cfg *config.MysqlConfig) (string, error) {
 	}
 
 	// Support TLS connection
-	if cfg.TLS != nil {
+	if cfg.TLSConfig != nil {
 		mysqlCfg.TLSConfig = "custom"
-		tls, err := cfg.TLS.Client()
+		tls, err := cfg.TLSConfig.Client()
 		if err != nil {
 			return "", err
 		}
 
 		if err := mysql.RegisterTLSConfig("custom", tls); err != nil {
 			return "", err
 		}
-	}
+	} else if cfg.TLS != "" { // If no custom config is specified, use tls parameter if it is set
+        mysqlCfg.Params = map[string]string{"tls": cfg.TLS}
```

Comment:
Code needs format.

Reply:
fixed

Reply:
You can refer to this https://github.com/go-sql-driver/mysql/issues/899。

Use mysql config's `TLSConfig`.

Reply:
Fixed :)

Reply:
@gaius-qi If everything is ok, could this be included in 2.0.4?

Reply:
> @gaius-qi If everything is ok, could this be included in 2.0.4?

Fix the lint workflow and the code implementation should be fine.

Reply:
@gaius-qi  Thanks, it should be fixed now.